sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/DocumentTranslationClientLiveTests.cs

```diff
@@ -33,19 +33,38 @@ public void ClientCannotAuthenticateWithFakeApiKey()
         [RecordedTest]
         public async Task GetDocumentFormatsTest()
         {
-            var client = GetClient();
+            DocumentTranslationClient client = GetClient();
 
             var documentFormats = await client.GetDocumentFormatsAsync();
+
             Assert.GreaterOrEqual(documentFormats.Value.Count, 0);
+            foreach (FileFormat fileFormat in documentFormats.Value)
+            {
+                Assert.IsFalse(string.IsNullOrEmpty(fileFormat.Format));
+                Assert.IsNotNull(fileFormat.FileExtensions);
+                Assert.IsNotNull(fileFormat.FormatVersions);
+            }
         }
 
         [RecordedTest]
         public async Task GetGlossaryFormatsTest()
         {
-            var client = GetClient();
+            DocumentTranslationClient client = GetClient();
 
             var glossaryFormats = await client.GetGlossaryFormatsAsync();
+
             Assert.GreaterOrEqual(glossaryFormats.Value.Count, 0);
+            foreach (FileFormat glossaryFormat in glossaryFormats.Value)
+            {
+                Assert.IsFalse(string.IsNullOrEmpty(glossaryFormat.Format));
+                Assert.IsNotNull(glossaryFormat.FileExtensions);
+                Assert.IsNotNull(glossaryFormat.FormatVersions);
+
+                if (glossaryFormat.Format == "XLIFF")
```

Comment:
just curious - does default version only come back for XLIFF?

Reply:
yes. for document format they don't return it, and in glossaries, TSV doesn't have versions